src/services/CertificateGenerationService.ts

```diff
@@ -49,42 +49,58 @@ class CertificateGenerationService {
      */
     public async generateCertificate(testResult: any): Promise<IGeneratedCertificateResponse> {
         const config: IMOTConfig = this.config.getMOTConfig();
+        const iConfig: IInvokeConfig = this.config.getInvokeConfig();
         const testType: any = testResult.testTypes;
-        const payload: any = await this.generatePayload(testResult);
+        const payload: any = JSON.stringify(await this.generatePayload(testResult));
```

Comment:
can be a string type now. Should it be, though?

Reply:
Agreed, will change that.